### ISSUE TITLE
Is HDV within the scope of the project?

### ISSUE THREAD

**Issue by kieranjol** [mipops/dvrescue#53]
Just checking, I wasn't clear from the readme if HDV is in the roadmap.

**Comment by dericed** [comment]
do you have samples of hdv captured from tape?

**Comment by digitensions** [comment]
I have some from my JVC camera, personal collection if needed.

**Comment by kieranjol** [comment]
Joanna's might be easier to share - i have some samples alright - FCP 7 captures, but I'd have to get clearance.

**Comment by digitensions** [comment]
Same, FCP7 firewire captures of progressive HDV. Happy to drop some over, I have clearance for it.

**Comment by kieranjol** [comment]
I was given permission to share some of these but they can't be made publicly available.

**Comment by JeromeMartinez** [comment]
"this" HDV is https://en.wikipedia.org/wiki/HDV#Specifications and actually it has no link (not a single bit is same) with DV despite the name.

Analysis is completely different. We could reuse the same ideas and the same user interface, but we need to implement all the MPEG Video logic instead of DV logic. There is already a lot in MediaInfo library from another project (e.g. we can check if headers or macroblocks are coherent), but there is still some work for having a good MPEG Video related report.

**Comment by kieranjol** [comment]
Imagine how bananas it is when you have both hdv and DV content on the same
tape. We encountered this a while back.

**Comment by JeromeMartinez** [comment]
@kieranjol I wonder how it is stored on the tape... the file you provided is a "classic" MPEG-2 Video stream in MOV, with Final Cut metadata. As far as I know MPEG-2 Video could not be as is on a tape (no small fixed packetization), Wikipedia says that is uses to be MPEG-TS (more logical) so the curating would be on MPEG-TS.

Anyway, far from this project (currently) focused on DV packetization.

**Comment by kieranjol** [comment]
Good point. The file I provided was from a vendor using FCP 7. Based on
tests, you could only really get FCP to capture via firewire with one
specific setting, which is what was used. Some sort of HDV preset. Sounds
like reencoding could be happening?

**Comment by JeromeMartinez** [comment]
> Sounds like reencoding could be happening?

No way to know if there was a reencoding. Probably a transwrapping (from MPEG-TS to MOV?).

> based on tests, you could only really get FCP to capture via firewire with one
specific setting, which is what was used.

Here it seems that FCP does the "rescue" of the file, but no idea what it would do in case of corrupted content, you have to rely on FDCP.
Without having the raw content from the tape, I doubt that a "rescue" software would be useful (in addition to the count of HDV files vs DV files, which puts a limit to the development acceptable cost).

**Comment by kieranjol** [comment]
Hi - I just shared a DVHSCap with Jerome and Dave there - this should be ore of a stream grab I hope. It's Mpeg-TS.

It's probably still out of scope but thought I'd share a better example of the format.

**Comment by JeromeMartinez** [comment]
@kieranjol right still out of scope because it is also MPEG-2 Video instead of DV... I would love to work on a MPEG-2 repair tool, but current sponsoring is not for that...

**Comment by dericed** [comment]
there are some possible side effects of the project that could help HDV but the grant is written for DV.
